inscriptions/ISic001401.xml

```diff
@@ -197,15 +197,17 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/><persName type="attested"><name type="nomen">Βετία</name> <space quantity="1" unit="character"/> <name type="cognomen" cert="low">Κο<choice><corr>ρν</corr><sic>μ</sic></choice>η
-                    <lb n="2" break="no"/>λία</name></persName> <space quantity="1" unit="character"/> εὐσεβεστ<choice><corr>ά</corr><sic>ε</sic></choice>
-                    <lb n="3" break="no"/>τ<choice><corr>η</corr><sic>α</sic></choice> <space quantity="1" unit="character"/> ἔζησε <g ref="#interpunct">·</g> ἔτ<choice><corr>η</corr><sic>π</sic></choice>
+                    <lb n="1"/><persName type="attested"><name type="nomen">Βετία</name> <space quantity="1" unit="character"/> <name type="cognomen" cert="low">Κορνη
+                    <lb n="2" break="no"/>λία</name></persName> <space quantity="1" unit="character"/>  <choice><orig>εὐσεβεστέ
+                    <lb n="3" break = "no"/>τα</orig><reg>εὐσεβεστάτη</reg></choice> <space quantity="1" unit="character"/> ἔζησε <g ref="#interpunct">·</g> ἔτη
```

Comment:
seems to be extra space around the = in the break="no", which will therefore not validate I think.